files/es/web/api/filereader/readasarraybuffer/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: FileReader.readAsArrayBuffer()
+slug: Web/API/FileReader/readAsArrayBuffer
+tags:
+  - API
+  - DOM
+  - File API
+  - FileReader
+  - Files
+  - Method
+  - Reference
+  - readAsArrayBuffer
+browser-compat: api.FileReader.readAsArrayBuffer
+translation_of: Web/API/FileReader/readAsArrayBuffer
+---
+{{APIRef("File API")}}
+
+El método **`readAsArrayBuffer()`** de la interfaz {{domxref("FileReader")}}
+se utiliza para comenzar a leer el contenido especificado en {{domxref("Blob")}}
+o {{domxref("File")}}. Cuando la operación de lectura finaliza,
+el método {{domxref("FileReader.readyState","readyState")}} se convierte en `DONE`,
```

Comment:
No es un método. Es una propiedad.

```suggestion
la propiedad {{domxref("FileReader.readyState","readyState")}} se convierte en `DONE`,
```

Reply:
The [FileReader](https://developer.mozilla.org/en-US/docs/Web/API/FileReader) interface's readAsArrayBuffer() method is used to start reading the contents of a specified [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) or [File](https://developer.mozilla.org/en-US/docs/Web/API/File).

@davbrito según yo si es metodo, donde dice propiedad?

https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsArrayBuffer